lxd/api_cluster.go

```diff
@@ -2019,60 +2019,126 @@ func clusterCertificatePut(d *Daemon, r *http.Request) response.Response {
 		return response.BadRequest(fmt.Errorf("Private key must be base64 encoded PEM key: %w", err))
 	}
 
+	err = updateClusterCertificate(r.Context(), d, r, req)
+	if err != nil {
+		return response.SmartError(err)
+	}
+
+	requestor := request.CreateRequestor(r)
+	s.Events.SendLifecycle(projectParam(r), lifecycle.ClusterCertificateUpdated.Event("certificate", requestor, nil))
+
+	return response.EmptySyncResponse
+}
+
+func updateClusterCertificate(ctx context.Context, d *Daemon, r *http.Request, req api.ClusterCertificatePut) error {
+	revert := revert.New()
+	defer revert.Fail()
+
+	newClusterCertFilename := shared.VarPath("cluster.crt.new")
```

Comment:
Can `cluster.crt.new` be a constant please so we can easily find how its related/used with the auto renewal function.